addons/rose/addon/components/rose/form/checkbox/index.stories.mdx

```diff
@@ -78,6 +78,7 @@ Renders a form input checkbox.
   @error={{false}}
   @disabled={{false}}
   @inline={{false}}
+  @outline={{false}}
```

Comment:
Outline shouldn't be a component attribute.  Just make it easy to toggle with CSS in case we decide not to use it for dropdown items.

This also does not apply to the base checkbox.  The base checkbox must always have an outline.